### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by cs-harsh-sharma**
#### Descriptions:

PMDB Link: https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=24348

UTCs:
Connector installation verified.
Connector logo verified.
Actions and Playbook list verified.
Playbook tags verified.
All Playbook are in info mode verified.
All Playbook are in inactive mode verified.

Verified all actions.

### CODE REVIEW

File: opswat-metadefender-core/operations.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+def submit_file(config, params):
+    opswat = Opswat(config)
+    try:
+        submit_type = params.get('submit_type')
+        submit_mode = params.get('submit_mode')
+        if submit_type == 'File Download URL':
+            if submit_mode == 'Synchronous Mode':
+                other_attributes = params.get('other_attributes')
+                payload = {
```
Comment (cs-omkar-kumbhar) [inline opswat-metadefender-core/operations.py]: write a common function to generate payload, it can be used in all 4 conditions

Comment (cs-harsh-sharma) [reply opswat-metadefender-core/operations.py]: done

File: opswat-metadefender-core/info.json [hunk 2]
```diff
@@ -0,0 +1,873 @@
+{
+  "name": "opswat-metadefender-core",
+  "label": "OPSWAT MetaDefender Core",
+  "description": "OPSWAT MetaDefender Core prevents malicious file uploads on web applications that bypass sandboxes and other detection-based security solutions. This connector facilitates operations to Submit File, Get Hashcode Reputation, Download Sanitized Files.",
+  "category": "Threat Intelligence",
```
Comment (cs-prerna-joshi) [inline opswat-metadefender-core/info.json]:
```suggestion
  "category": "Malware Analysis",
```

File: opswat-metadefender-core/info.json [hunk 3]
```diff
@@ -0,0 +1,873 @@
...
+      "parameters": [
+         {
+          "title": "Submission Type",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "select",
+          "options": [
+            "File Download URL",
+            "Through Attachments"
```
Comment (cs-prerna-joshi) [inline opswat-metadefender-core/info.json]:
```suggestion
            "FortiSOAR File/Attachments IRI"
```

**Pull request merged**